docs/DeveloperGuide.adoc

```diff
@@ -782,13 +782,16 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
```

Comment:
Or you can also consider stating "veterinarians and veterinary technicans" beside "target user profile" to be clear and upfront about our target user profile.